### ISSUE TITLE
[3.1.4] a11y subtitles partially not working on Firefox

### ISSUE THREAD

**Issue by pkra** [mathjax/MathJax#2679]
### Issue Summary

On Firefox, after activating the accessibility extension and starting to explore an expression, the subtitles remain (mostly) "empty", i.e., the box for them appears but it doesn't always fill with content.
  
### Steps to Reproduce:

1. open mathjax.org#demo on FF
2. activate the accessibility extension
3. explore an expression

E.g., in the block expression, x will not show, equals will, fraction parts will show "Numerator" and "Denominator" but not the contents.

### Technical details:

* MathJax Version: 3.1.4
* Client OS: ubuntu 20.10
* Browser: Firefox 88

**Comment by zorkow** [comment]
The `aria-label` is computed correctly. For some reason the individual speech attributes are only partially filled in.
